xray/capture_test.go

```diff
@@ -20,91 +20,91 @@ import (
 )
 
 func TestSimpleCapture(t *testing.T) {
-	TestDaemon.Reset()
-	ctx, root := BeginSegment(context.Background(), "Test")
-	err := Capture(ctx, "TestService", func(ctx1 context.Context) error {
-		ctx = ctx1
-		defer root.Close(nil)
+	ctx, td := NewTestDaemon()
+	defer td.Close()
+
+	ctx, root := BeginSegment(ctx, "Test")
+	err := Capture(ctx, "TestService", func(context.Context) error {
+		root.Close(nil)
 		return nil
 	})
 	assert.NoError(t, err)
 
-	s, e := TestDaemon.Recv()
-	assert.NoError(t, e)
-	assert.Equal(t, "Test", s.Name)
-	assert.Equal(t, root.TraceID, s.TraceID)
-	assert.Equal(t, root.ID, s.ID)
-	assert.Equal(t, root.StartTime, s.StartTime)
-	assert.Equal(t, root.EndTime, s.EndTime)
-	assert.NotNil(t, s.Subsegments)
-	subseg := &Segment{}
-	assert.NoError(t, json.Unmarshal(s.Subsegments[0], &subseg))
-	assert.Equal(t, "TestService", subseg.Name)
-}
-
-func TestCaptureAysnc(t *testing.T) {
```

Comment:
Why have we removed captureAsync test ?

Reply:
It's TestCapture A**ys**nc, not TestCaptureA**sy**nc.
and there is already TestCaptureA**sy**nc in the end of this file.